Lib/Dojo.as

```diff
@@ -166,17 +166,12 @@ class TMDojo
         bool hudOff = false;
 
         if (app.CurrentPlayground !is null && app.CurrentPlayground.Interface !is null) {
-            if (!UI::IsGameUIVisible()) {
-                hudOff = true;
```

Comment:
Is this not needed anywhere? Looks like something that we might still want to set, even if the way to get the time is the same for both cases.